src/registry/MergedRegistry.ts

```diff
@@ -8,17 +8,28 @@ import { IRegistry, RegistryContent, RegistryType } from './IRegistry.js';
 
 export interface MergedRegistryOptions {
   registries: Array<IRegistry>;
+  chainMetadataOverrides?: ChainMap<ChainMetadata>;
```

Comment:
should this be `ChainMap<Partial<ChainMetadata>>` as well?

Reply:
yes!